datafusion/physical-expr/src/sort_expr.rs

```diff
@@ -75,21 +79,57 @@ impl PhysicalSortExpr {
 }
 
 /// Represents sort requirement associated with a plan
+///
+/// If the requirement is *exact*
+/// ([`PhysicalSortRequirement::new_exact`]), then the sort
+/// requirement will only be satisfied if it matches both the
+/// expression *and* the sort options.
+///
+/// If the requirement is *`expr_only`
+/// ([`PhysicalSortRequirement::new_expr_only`]) then only the expr
+/// must match, to satisfy the requirement.
+///
```

Comment:
It seems that we now use `PhysicalSortRequirement::new` for initialization for both cases. These comments seem remainder for old API.

Reply:
👍  that is a good spot -- fixed in 1db636ada